applications/DeepSpeed-Chat/training/step1_supervised_finetuning/prompt_eval.py

```diff
@@ -194,7 +195,7 @@ def prompt_eval(args, model_baseline, model_fintuned, tokenizer, device,
 def main():
     args = parse_args()
 
-    device = torch.device("cuda:0")
+    device = torch.device(get_accelerator().device_name()+":0")
```

Comment:
```suggestion
    device = torch.device(get_accelerator().device_name(0))
```

Reply:
Thanks, I have updated the code with your suggestions.